lib/cash-out/cash-out-atomic.js

```diff
@@ -62,7 +63,10 @@ function preProcess (t, oldTx, newTx, pi) {
     .then(updatedTx => {
       if (updatedTx.status !== oldTx.status) {
         const isZeroConf = pi.isZeroConf(updatedTx)
-        if (wasJustAuthorized(oldTx, updatedTx, isZeroConf)) pi.sell(updatedTx)
+        if (wasJustAuthorized(oldTx, updatedTx, isZeroConf)) {
+          pi.sell(updatedTx)
+          pi.notifyOperator(updatedTx, { isRedemption: false })
```

Comment:
I think this should be handled as a promise, including catching errors or letting them bubble up. Otherwise we could get uncaught promise errors.